wiki/Other-resources.md

```diff
@@ -1,2 +1,2 @@
 * Hand tracking OSC for VRChat with ALVR support: https://github.com/A3yuu/FingerTruckerOSC
-* ALVR in the browser using WebXR and WebCodecs: https://github.com/rinsuki-lab/ALVR/tree/research/alvr-web
\ No newline at end of file
+* ALVR in the browser using WebXR and WebCodecs: https://github.com/rinsuki-lab/ALVR/tree/research/alvr-web
```

Comment:
This seems like a spurious change but i guess you can keep it in since the new line was missing

Reply:
This was an automated commit a GitHub action made when I prepared the fork, I'm not sure why its here.